ecs_composex/compose/compose_services/__init__.py

```diff
@@ -754,6 +754,25 @@ def ulimits(self) -> Union[list, Ref]:
 
         return rendered_limits if rendered_limits else NoValue
 
+    @property
+    def stop_grace_period(self):
+        """
+        Method to import and determine StopTimeout
+        """
+        __stop_grace_period = set_else_none("stop_grace_period", self.definition)
+        if not __stop_grace_period:
+            return NoValue
+        if not isinstance(__stop_grace_period, (int, str, float)):
+            raise TypeError(self.name)
+        stop_timeout = import_time_values_to_seconds(__stop_grace_period)
```

Comment:
```suggestion
        stop_timeout = import_time_values_to_seconds(__stop_grace_period, 120)
```
the function `import_time_values_to_seconds` already handles to set a max value. Given ECS max value is 120, I'd set it there? Or did you handle it below to warn the user?

Reply:
I did handle it below so in order to warn the user of too short/long, but I'm happy to do it without the warning if you prefer?